src/Delorean/Local/QQ.hs

```diff
@@ -40,3 +41,6 @@ qmoh = qmaybe (\s -> readMaybe (T.unpack s) >>= minuteOfHourFromInt)
 
 qsom :: QuasiQuoter
 qsom = qmaybe (\s -> readMaybe (T.unpack s) >>= secondOfMinuteFromInt)
+
+qdom :: QuasiQuoter
```

Comment:
Slightly overloaded use of `m` here, above it refers to minute and here it refers to month